src/App.tsx

```diff
@@ -8,7 +8,52 @@
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './services/todos';
+
+import { Todo } from './types/Todo';
+import { Visible } from './types/Visible';
+
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [isListLoading, setIsListLoading] = useState<boolean>(false);
+  const [listErrorMessage, setListErrorMessage] = useState<string>('');
+
+  const [visible, setVisible] = useState<Visible>('all');
+  const [filterBy, setFilterBy] = useState<string>('');
+
+  const [activeTodo, setActiveTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    setIsListLoading(true);
+
+    getTodos()
+      .then(data => {
+        setTodoList(data);
+      })
+      .catch(() => setListErrorMessage('Try again Later'))
+      .finally(() => {
+        setIsListLoading(false);
+      });
+  }, []);
+
+  const currentTodoList = todoList.filter(el => {
+    if (visible == 'all') {
+      return el.title.toLowerCase().includes(filterBy.toLowerCase()) && el;
```

Comment:
The condition `&& el` is unnecessary here. The filter function should return a boolean, so you can simplify this by just returning the boolean expression.